app/assets/javascripts/modules/collapsible_collection.js

```diff
@@ -96,30 +96,30 @@
     return selector;
   };
 
-  CollapsibleCollection.prototype.closeAll = function closeAll(event){
+  CollapsibleCollection.prototype.closeAll = function closeAll(){
     for (var section in this.collapsibles) {
+      // as $clickTarget is the whole h2 and is available, we just need to get the button
+      var $button = $(this.collapsibles[section].$clickTarget[0].querySelector('button'))
+
       this.collapsibles[section].close();
+      $button.attr('aria-expanded', 'false');
```

Comment:
Can we remove the changes in the `openAll` and `closeAll` functions and instead encapsulate the behaviour in `collapsible.js`?

i.e. `this.collapsibles[section].close()` should, inside its function definition, ensure that the button `aria-expanded` is `false`.

Reply:
moved into collapsible and trimmed down